### ISSUE TITLE
Allow to configure entrypoint sleep

### ISSUE THREAD

**Issue by gaby** [vishnudxb/redis-cluster#3]
Hello @vishnudxb 

We are having the same issue from #1 again, would it be possible for you to make the sleep configurable instead of hardcoded to 5secs. It can still default to that, but allow us to change it.

```
Status: Downloaded newer image for vishnunair/docker-redis-cluster:latest
Could not connect to Redis at localhost:6379: Connection refused
cluster_state:fail
cluster_slots_assigned:0
cluster_slots_ok:0
cluster_slots_pfail:0
cluster_slots_fail:0
cluster_known_nodes:1
cluster_size:0
cluster_current_epoch:0
cluster_my_epoch:0
cluster_stats_messages_sent:0
cluster_stats_messages_received:0
total_cluster_links_buffer_limit_exceeded:0
Redis Cluster is up and running!
```

I can do a PR for it, if needed. Thanks

**Comment by vishnudxb** [comment]
@gaby  Released, you can use version 1.0.9
```
steps:
- uses: vishnudxb/redis-cluster@1.0.9
  with:
    master1-port: 5000
    master2-port: 5001
    master3-port: 5002
    slave1-port: 5003
    slave2-port: 5004
    slave3-port: 5005
    sleep-duration: 5
```

**Comment by gaby** [comment]
@vishnudxb Thanks for the quick turn around!
